release/README.md

```diff
@@ -0,0 +1,78 @@
+# Release Action
+
+This action automates the manual steps needed to generate changelogs and push release tags.
+For more information on the overall flow of this ci image, take a look at the [elementary release process documentation](https://github.com/elementary/os/wiki/Release-Process).
+
+## Requirements
+
+This image is intended for use with elementary debian projects. There are a few requirements before getting started:
+
+  1. The project needs to have a deb-packaging branch with the necessary debian packaging for the project. Docs: [debian control](https://elementary.io/docs/code/getting-started#debian-control)
+  2. The project needs release information in an `appdata.xml` file. Docs: [appdata](https://elementary.io/docs/code/getting-started#appdata)
+
+### Environment Variables
+
+In order to create tags and push changes to various branches, the script needs a github token. Keep in mind, when using github workflows, the virtual environment [automatically comes with a generated github token secret](https://help.github.com/en/articles/virtual-environments-for-github-actions#github_token-secret).
+
+### Specifying a release branch name
+
+By default, this action will create and update a branch named 'stable' whenever a release is pushed. The branch name can be set via the `release_branch` input. Example:
+
+```yaml
+with:
+  release_branch: 'juno'
+```
+
+### Specifying a label
+
+By default, the examples check for a label called `Release` on the related pull request. This can be set in the workflow action by changing the following:
+
+```yaml
+# check for "Release" label:
+true == contains(join(github.event.pull_request.labels.*.name), 'Release')
+# check for "Example" label:
+true == contains(join(github.event.pull_request.labels.*.name), 'Example')
+
+```
+
+## Examples
+
+### Simple Example
+
+```yaml
+name: Release
+on:
+  pull_request:
+    branch: master
```

Comment:
oops wait this should be
```suggestion
    branches: master
```

Reply:
It looks like it can be either. I just tested both `branch` and `branches` in the helloworld project's ci file. I'm cool with changing it to `branches` though :+1: